ravenframework/Models/ModelPlugInFactory.py

```diff
@@ -42,7 +42,11 @@
 ## -> should we do this through some kind of a PluginLoader that registers the objects
 ##    in other factories?
 
-__basePluginClasses = {'ExternalModel':'ExternalModelPluginBase'}
+__basePluginClasses = {'ExternalModel':'ExternalModelPluginBase',
+                      'Code': 'CodePluginBase',
+                      'PostProcessor': 'PostProcessorPluginBase',
+                      'SupervisedLearning': 'SupervisedLearningPlugin',
```

Comment:
Thanks for the update here, it seems OutStreamPlot is missing here, could you update it?

Reply:
@wangcj05 I did not put it here because this is the "Model Plugin factory". Should it be listed here even if it is a "Plot plugin"?

Reply:
You are right, I did not realize that.